fcc-test-suite-template-master/index.html

```diff
@@ -39,11 +39,14 @@ <h1><center>Here's a time line of Dr. César's life:</center></h1>
             Works in the Civilian Conservation Corps, helping starving
             Americans. "I saw how food changed them", he said. "All of this left
             scars on me."
+            <strong></strong>
           </li>
           <li>
             <strong>1937</strong> - Finishes university and takes a job in the
             US Forestry Service
           </li>
+
```

Comment:
- [ ] Great comments 😃 , but as the previous reviewer said, as reviewers, we also focus on things the student did well on his code so it will be great if you highlight some of the work that the student did well. For example you may say something like `good job working on...` and add the comment 👍 

Eg. `Great comments ✨ however`, it only lacks a little bit of explanation and it can be improved 🚀